### PR TITLE
data: AbhayaShankar

### PR DESCRIPTION
**Pull request by AbhayaShankar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7296"></a>

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

**Comment by AbhayaShankar** [comment]
Thanks @Panquesito7 🚀

**Review by Panquesito7 — DISMISSED** [review]

File: data/AbhayaShankar.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      "url" : "https://www.linkedin.com/in/abhayashankar/",
+      "icon" : "FaLinkedin",
+      "color" : "#204ecf"
+      
+    },
+    {
+      "group" : "Socials",
+      "name" : "Abhaya : Discord",
+      "url" : "discordapp.com/users/Abhaya1289#3250",
+      "icon" : "FaDiscord",
```
Comment (SaraJaoude) [inline data/AbhayaShankar.json:68]: remove comma:
```suggestion
      "icon" : "FaDiscord"
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**